src/github-graphql.js

```diff
@@ -271,9 +271,12 @@ class GraphQLClient {
         data.repository.pullRequest.comments.nodes);
     } else {
       this.warningCount++;
-      console.log('warning: no available reaction data!',
+      if (DEBUG) {
+         console.log('warning: no available reaction data!',
         'owner:', owner, 'name:', name, 'pull request number:', number,
```

Comment:
Please add indentation here.

Reply:
Sorry, for making this kind of mistake.
Please review again.